.github/workflows/continuous-integration.yml

```diff
@@ -17,53 +17,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-13, macos-latest, windows-2022]
-        gfortran-version: [12, 13, 14]
-        gfortran-from: [system, conda]
+        rte-kernels: [default, accel]
         fpmodel: [DP, SP]
+        build-type: [Debug, RelWithDebugInfo]
```

Comment:
We test `RelWithDebugInfo` with no extra flags in `conda-style-builds.yml`. Instead of `Debug` with extra flags and `RelWithDebugInfo` without extra flags, I would test only `RelWithDebugInfo` with extra flags in this workflow.